lib/defaults.js

```diff
@@ -27,7 +27,7 @@
 	}
 
 	const cacheExtra = process.platform === 'win32' ? 'npm-cache' : '.npm';
-	const cacheRoot = process.platform === 'win32' ? process.env.APPDATA : home;
+	const cacheRoot = process.platform === 'win32' && process.env.APPDATA || home;
```

Comment:
It looks like this change should actually be made in the babylon template: https://github.com/kevva/npm-conf/blob/173207aa7fa69f7284208e33df2a3880e031a0f6/lib/make.js#L37

This file (`lib/defaults.js`) is actually generated output, and isolated changes will be overwritten the next time anyone runs `node lib/make.js` (which is the `prepublish` script, and thus explains why this change is not present in the most recent version published to npm). It seems weird to have this file in version control in the first place.